components/Buttons/examples/ButtonsShapesExampleViewController.m

```diff
@@ -59,7 +70,7 @@ - (void)viewDidLoad {
 
   MDCButton *containedButton = [[MDCButton alloc] init];
   [containedButton setTitle:@"Add To Cart" forState:UIControlStateNormal];
-  [MDCContainedButtonThemer applyScheme:buttonScheme toButton:containedButton];
+  [containedButton applyContainedThemeWithScheme:[self containerScheme]];
```

Comment:
Just realized containerScheme is defined as a property. Maybe use `self.containerScheme` here?

Reply:
`containerScheme` in this case is a method, so the `[self ...]` invocation is correct here.

Reply:
Ah - in that case this should use property notation.

Reply:
Done